packages/dev/core/src/Maths/math.vector.ts

```diff
@@ -297,16 +1938,88 @@ export class Vector2 {
      * @returns the current updated Vector2
      */
     public divideInPlace(otherVector: DeepImmutable<Vector2>): this {
-        return this.divideToRef(otherVector, this);
+        this.x = this.x / otherVector.x;
+        this.y = this.y / otherVector.y;
+        return this;
+    }
+
+    /**
+     * Updates the current Vector2 with the minimal coordinate values between its and the given vector ones
+     * @param other defines the second operand
+     * @returns the current updated Vector2
+     */
+    public minimizeInPlace(other: DeepImmutable<Vector2>): this {
+        return this.minimizeInPlaceFromFloats(other.x, other.y);
+    }
+
+    /**
+     * Updates the current Vector2 with the maximal coordinate values between its and the given vector ones.
+     * @param other defines the second operand
+     * @returns the current updated Vector2
+     */
+    public maximizeInPlace(other: DeepImmutable<Vector2>): this {
+        return this.maximizeInPlaceFromFloats(other.x, other.y);
+    }
+
+    /**
+     * Updates the current Vector2 with the minimal coordinate values between its and the given coordinates
+     * @param x defines the x coordinate of the operand
+     * @param y defines the y coordinate of the operand
+     * @returns the current updated Vector2
+     */
+    public minimizeInPlaceFromFloats(x: number, y: number): this {
+        if (x < this.x) {
+            this.x = x;

```

Comment:
this.x = math.min(x, this.x) ? and same below

Reply:
Fixed in [157fc18](https://github.com/BabylonJS/Babylon.js/pull/13699/commits/157fc18cd530c9949d920475cb52ee5670b09539)